### PR TITLE
Explicit tests

### PR DESCRIPTION
**Pull request by Jopyth**

### CODE REVIEW

File: length/tests/function_tests/test_sigmoid.py [hunk 1]
```diff
@@ -1,17 +1,27 @@
 import numpy as np
 
 from length import constants
```
Comment (Bartzi) [inline length/tests/function_tests/test_sigmoid.py]: I think we need to clean all imports once we are ready to remove code...

File: length/tests/optimizer_tests/test_sgd.py [hunk 2]
```diff
@@ -1,14 +1,25 @@
 import numpy as np
 
+from length.constants import DTYPE
```
Comment (Bartzi) [inline length/tests/optimizer_tests/test_sgd.py]: we need to make this consistent, I always used `constants.DTYPE` but here we only use `DTYPE`... I could live with only using `DTYPE`

File: length/tests/optimizer_tests/test_sgd.py [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
 from length.optimizers.sgd import SGD
 from length.tests.gradient_checker import assert_allclose
 
 
 def test_sgd():
-    learning_rate = 0.001
+    learning_rate = 0.0017
```
Comment (Bartzi) [inline length/tests/optimizer_tests/test_sgd.py:13]: nice tweak^^

**Review by Bartzi — APPROVED** [review]
:grey_question:

File: length/tests/optimizer_tests/test_sgd.py (hunk 3 above)
Comment (Jopyth) [reply length/tests/optimizer_tests/test_sgd.py:13]: :+1:

**Pull request merged**